frontend/document/pdfdocument.lua

```diff
@@ -36,12 +33,11 @@ function PdfDocument:init()
     if not ok then
         error(self._document)  -- will contain error message
     end
+    self.epub_font_size = self:getKoptFontSize()
```

Comment:
Looks ok as a whole.
Except for the namings which make it a bit hard to follow :)
`self:getKoptFontSize()`  sounds generic, but it returns something that matters only for reflowable documents.
`epub_font_size` sounds obvious, but might be better to name it `reflowable_font_size` and stay with "reflowable" in the related functions and variables names, like the nicely named `self.is_reflowable` and `document:isDocumentReflowable()`

Reply:
Should be better now.